### PR TITLE
SLVS-1621 Clean up custom SSE implementation for taints

### PR DESCRIPTION
**Pull request by georgii-borovinskikh-sonarsource**
[SLVS-1621](https://sonarsource.atlassian.net/browse/SLVS-1621)

[SLVS-1621]: https://sonarsource.atlassian.net/browse/SLVS-1621?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/SonarQube.Client/Models/ServerSentEvents/SSEStreamReader.cs [hunk 1]
```diff
@@ -52,8 +52,6 @@ internal class SSEStreamReader : ISSEStreamReader
         private readonly IDictionary<string, Type> eventTypeToDataTypeMap = new Dictionary<string, Type>
```
Comment (gabriela-trutan-sonarsource) [inline src/SonarQube.Client/Models/ServerSentEvents/SSEStreamReader.cs:52]: Maybe we can run Resharper's code cleanup + reformat on this class.
It has some unused usings

File: src/ConnectedMode/ServerSentEvents/SSESessionFactory.cs [hunk 2]
```diff
@@ -18,213 +18,195 @@
...
 using SonarLint.VisualStudio.ConnectedMode.ServerSentEvents.QualityProfile;
-using SonarLint.VisualStudio.ConnectedMode.ServerSentEvents.Taint;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.ServerSentEvents;
 using SonarQube.Client;
 using SonarQube.Client.Models.ServerSentEvents;
 using SonarQube.Client.Models.ServerSentEvents.ClientContract;
 
-namespace SonarLint.VisualStudio.ConnectedMode.ServerSentEvents
+namespace SonarLint.VisualStudio.ConnectedMode.ServerSentEvents;
```
Comment (gabriela-trutan-sonarsource) [inline src/ConnectedMode/ServerSentEvents/SSESessionFactory.cs:31]: Maybe we can run Resharper's code cleanup + reformat on this class.

**Review by gabriela-trutan-sonarsource — APPROVED** [review]
Just a minor feedback as part of CaYC

File: src/SonarQube.Client/Models/ServerSentEvents/SSEStreamReader.cs (hunk 1 above)
Comment (georgii-borovinskikh-sonarsource) [reply src/SonarQube.Client/Models/ServerSentEvents/SSEStreamReader.cs:52]: Sorry, I will skip this. Need to move on.

File: src/ConnectedMode/ServerSentEvents/SSESessionFactory.cs (hunk 2 above)
Comment (georgii-borovinskikh-sonarsource) [reply src/ConnectedMode/ServerSentEvents/SSESessionFactory.cs:31]: Sorry, I will skip this. Need to move on.

**Pull request merged**